app/models/order.rb

```diff
@@ -15,7 +17,7 @@ def add_line_items_from_cart(cart)
      line_items << item
     end
  end
-
+ 
```

Comment:
why this change?

Reply:
An additional space was added by mistake